ui-tests/src/test/java/org/openmrs/reference/DeleteAppointmentBlockTest.java

```diff
@@ -34,6 +34,7 @@ public void setUp() throws Exception {
      * @verifies creation and delete of an appointment
      */
     @Test
+    // Rebuild the tests present in the module
```

Comment:
thanks @kdaud  this looks more of a comment than a fix!

Reply:
> this looks more of a comment than a fix!

I agree. 

However, just as the comment states am investigation the behavior of all the resurrected tests present in master to make sure if there is any of them breaking things with CI its identifiable and reverted for further investigation

Thanks for the review @HerbertYiga. After fixing issues with this particular test `DeleteAppointmentBlocktest` I will then migrate the ticket to CodeReview status for reviewers to take a glace at the fix